### PR TITLE
Feat/adding json server

### PR DESCRIPTION
**Pull request by tauantcamargo**

### CODE REVIEW

**Review by izass — APPROVED** [review]

**Review by Isaac-alencar — APPROVED** [review]
Looks good to me! :)

**Review by Isaius — APPROVED** [review]
ship it.

File: idega/fake-backend/db.json [hunk 1]
```diff
@@ -0,0 +1,274 @@
...
+  "wines": [
+    {
+      "id": 1,
+      "name": "Vinho",
+      "year": 2017,
+      "wineryOfOrigin": "Bertoletti",
+      "mainImage": "https://imageio.forbes.com/specials-images/imageserve//6276c7a49c222289a2752bc2/0x0.jpg?format=jpg",
+      "sellingPrice": 300.0,
+      "quantity": 10,
+      "moreDetails": {
```
Comment (BrunoViveiros) [inline idega/fake-backend/db.json:11]: I think we don't need to repeat
`id, name, year, winery_of_origin , main_image , quantity` on moreDetails

**Review by Isaac-alencar — APPROVED** [review]
lgtm

**Pull request closed without merging**